packages/Python/lldbsuite/test/lang/swift/swift_version/TestSwiftVersion.py

```diff
@@ -0,0 +1,93 @@
+# TestSwiftVersion.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test that LLDB can debug  code generated by the Swift compiler for different versions of the language
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+import lldbsuite.test.decorators as decorators
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+
+def execute_command(command):
+    # print '%% %s' % (command)
+    (exit_status, output) = commands.getstatusoutput(command)
+    # if output:
+    #     print output
+    # print 'status = %u' % (exit_status)
+    return exit_status
+
+class TestSwiftVersion(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    @decorators.skipUnlessDarwin
+    @decorators.swiftTest
+    def test_cross_module_extension(self):
+        """Test that LLDB can debug different Swift language versions"""
+        self.buildAll()
+        self.do_test()
+
+    def setUp(self):
+        TestBase.setUp(self)
+
+    def buildAll(self):
+        execute_command("make everything")
+
+    def do_test(self):
+        """Test that LLDB can debug different Swift language versions"""
+        def cleanup():
+            execute_command("make cleanup")
+        self.addTearDownHook(cleanup)
+
+        exe_name = "main"
+        exe_path = os.path.join(os.getcwd(), exe_name)
+
+        tests = [
+          { 'file' : "mod3.swift",
+            'source_regex' : "break 3",
+            'expr' : "S3().i",
+            'substr' : "3" },
+          { 'file' : "mod4.swift",
+            'source_regex' : "break 4",
+            'expr' : "S4().i",
+            'substr' : "4" }
+        ]
+
+        # Create the target
+        target = self.dbg.CreateTarget(exe_path)
+        self.assertTrue(target, VALID_TARGET)
+
+        for t in tests:
+          source_name = t['file']
+          source_spec = lldb.SBFileSpec(source_name)
+
+          breakpoint = target.BreakpointCreateBySourceRegex(t['source_regex'], source_spec)
+          self.assertTrue(breakpoint.GetNumLocations() > 0, VALID_BREAKPOINT)
```

Comment:
Since you're running a couple of tests in a for loop, if one of the instances fails three will be no breadcrumbs telling us which one.  Can you replace VALID_BREAKPOINT with some string that indicates while breakpoint you were trying to set?